### DISCUSSION TITLE
Activate strelka module after installation

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#3431] [Unsupported Versions] [answered]
Hello all,

SO Standalone 2.3.30
8 cores
16 GB Vram
2 TB diskspace

I installed SO without activating strelka, now that i'm sure i have enough diskspace i want to activate strelka.

I am sure the question has already been asked, but I cannot find it again.

I known that you must change in global.sls the setting enabled under strelka, but do I have to do anything else?

Regards
Bart

**Comment by weslambert** [comment, marked as answer]
You can enable Strelka within `/opt/so/saltstack/local/pillar/global.sls` (strelka.enabled: 1) on the manager.  Once you've done that, you can go to the downstream node (or on the same node if it is a standalone) and run `so-strelka-start`.
